src/stores/store.ts

```diff
@@ -269,15 +269,25 @@ export const useStore = defineStore('store', {
         this.setNetworkMessage('dashboard.verifyingNet');
 
         await client.startFarming(this.plotPath, this.plotSizeGB);
-
         // TODO: consider moving logging to client.ts
-        util.infoLogger('PLOTTING PROGRESS | farmer started');
+        util.infoLogger('Farmer started');
+      } catch (error) {
```

Comment:
first of all, this catch is printing the error of "farmer start error". 
So, the scope of the `try` block should only encapsulate the `farmer start` functionality. 

In the previous version, where `try` block is containing the whole logic, the error was happening after the farmer start is successful, yet the printed error is `farmer start error`, this is confusing. 

Hence, I narrowed down the scope of the `try` block here.